src/tools/csg_fmatch.cc

```diff
@@ -269,11 +283,16 @@ void CGForceMatching::EvalConfiguration(Topology *conf, Topology *conf_atom)
     if(_has_existing_forces) {
         if(conf->BeadCount() != _top_force.BeadCount())
             throw std::runtime_error("number of beads in topology and force topology does not match");
-        for(int i=0; i<conf->BeadCount(); ++i) {
+        for(int i=0; i<conf->BeadCount(); ++i) {                
+//            cout << "conf->getBead(" << i << ")->getPos(): " << conf->getBead(i)->getPos() << endl;                
+//            cout << "_top_force->getBead(" << i << ")->getPos(): " << _top_force.getBead(i)->getPos() << endl; 
             conf->getBead(i)->F() -= _top_force.getBead(i)->getF();
             vec d = conf->getBead(i)->getPos() - _top_force.getBead(i)->getPos();
-            if(abs(d) > 1e-6)
-                throw std::runtime_error("One or more bead positions in mapped and reference force trajectory differ by more than 1e-6");
+//            cout << "vec d of bead " << i << ": " << d << "abs(d): " << abs(d) << endl; 
+            if(abs(d) > 1e-5)//altered to 1e-5, otherwise it can be a too strict criterion
```

Comment:
I think we should make 1e-5, a parameter, which is read from the xml file and use 1e-5 the default!